app/uk/gov/hmrc/tariffclassificationfrontend/views/partials/advanced_search_results.scala.html

```diff
@@ -90,11 +91,12 @@
      <span id="advanced_search_results-row-@{index}-keywords">@result.c.keywords.mkString(", ")</span>
     </div>
 
+    @*Attached Images*@
```

Comment:
I would prefer to have a component called display_attached_images or something  similar  rather than a comment :)

Reply:
?  I added the comment to match the others for the different items in the search results

Reply:
It was a suggestion/preference, easier to test and maintain but feel free to resolve conversation.